### PR TITLE
Create Hosts-Guideline.md

### PR DESCRIPTION
**Pull request by slashmili**
a page that we can refer the host to read

### CODE REVIEW

File: Hosts-Guideline.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Elixir community in Berlin
+
+## When is it happening?
+
+Every 2nd Thursday of month usually between 7-10 pm
+
+## What is required to be a host?
+
+
+
```
Comment (duksis) [inline Hosts-Guideline.md]: what are these empty lines for?

Comment (slashmili) [reply Hosts-Guideline.md]: It's me entering too much 🤣

File: Hosts-Guideline.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## Not required but highly appreciated
+
+* :beers: Drinks alcoholic and/or non-alcoholic
+* :pizza: food 
+* :briefcase: Swag bag for speakers(if you are not sure, contact us and you can sponser giveaway Elixir Berlin swag with your compnay name on it!)
+
+## Looks good?
+
+Great! head to [issue](https://github.com/elixir-berlin/planning/issues) section, we usually try to plan ahead our next 3 month
```
Comment (duksis) [inline Hosts-Guideline.md]: > 3 month**s**

instead of 

> 3 month

**Comment by slashmili** [comment]
Do you think it's good enough to merge ?

**Pull request merged**